test/unit/v1.0.0/converter-v1-to-v21.test.js

```diff
@@ -59,7 +59,7 @@ describe('v1.0.0 to v2.1.0', function () {
                             header: [],
                             url: {
                                 raw: '',
-                                variable: [{ id: 'foo', value: 'bar' }]
+                                variable: [{ key: 'foo', value: 'bar' }]
```

Comment:
saw this here and in https://github.com/postmanlabs/postman-collection-transformer/pull/330. Was this a bug?